package.json

```diff
@@ -111,6 +112,7 @@
     "inquirer": "^8.0.0",
     "leasot": "^11.5.0",
     "lerna": "^4.0.0",
+    "lighthouse": "^7.2.0",
```

Comment:
If this is still a work in progress maybe we can leave these dependencies out of `package.json`. I'm also wondering if this is something we want to do in CI, maybe there's a github action / integration that'll provide this without the extra dependencies?

Reply:
Pulled them out in the latest commit